service/src/main/resources/HTMLTemplate.html

```diff
@@ -946,8 +1206,9 @@ <h3 class="heading heading--two">Child {{ counter }}</h3>
           <div class="data-list__row">
             <dt class="data-list__title">Social worker's telephone number</dt>
             <dd class="data-list__definition">
-              {% if child.value.socialWorkerTel is not empty %}
-                {{ child.value.socialWorkerTel }}
+              {% if child.value.party.socialWorkerTelephoneNumber is not empty
```

Comment:
seeing how this and socialWorkerName is set up now, I think it might be better to make better use of the telephoneNumber type. We could have socialWorkerDetails as type telephoneNumber. Then contactDirection = socialWorkerName, and telephoneNumber = telephoneNumber. Links the two pieces of data and then looks a bit nice then socialWorkerTelephoneNumber.TelephoneNumber

Reply:
this will be changes in the java code and in the migration tool

Reply:
let me know how this changes the UI please

Reply:
I agree and like this change Sam but this would have an affect on respondents TelephoneNumber, as in it would add an extra field. We would need to add an additional Telephone type just for children to accomplish this.